src/test/java/net/openhft/chronicle/core/JvmTest.java

```diff
@@ -271,6 +272,7 @@ static class ClassD extends ClassC {
     }
 
     interface InterfaceWithDefaultMethod {
+        @SuppressWarnings("EmptyMethod")
```

Comment:
should this be on helloDefault?

Reply:
The warning is that all implementations are empty.